packer/docs/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
```

Comment:
please reformat this file, and remove the mixed tabs and spaces. we should be using spaces. did you edit the file in vscode?

Reply:
formatted